components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3366,6 +3367,11 @@ private void doPatchGroup(String groupId, String currentGroupName, Map<String, L
                 temporaryMembers.clear();
 
                 for (String member : deletedMembers) {
+                    if (addedMembers.isEmpty() && (member == null || member.isEmpty())) {
```

Comment:
let's refactor this.

eg:
```java
for (String member : deletedMembers) {
    if (addedMembers.isEmpty() && StringUtils.isBlank(member)) {
        throw new BadRequestException(ResponseCodeConstants.INVALID_VALUE);
    } 

    if (StringUtils.isNotBlank(member)) {
        String username = UserCoreUtil.addDomainToName(UserCoreUtil.removeDomainFromName(member), userStoreDomainForGroup);
        temporaryMembers.add(username);
    }
}
```

Reply:
Refactored in this [fedd887](https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/478/commits/fedd8873c8f5897f92c93a9d7f57cb4815026047)